content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,256 @@
+|-----------|-------------|---------|
+| Access | Controls cluster access. Private access means that nodes and pods are isolated from the internet. | Private |
+| Identity Provider | You can use Palette as the IDP or a third-party application as the IDP. | Palette |
+
+#### Internal Manifest Parameters
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| namespace | The Kubernetes namespace to install the dashboard. | `kubernetes-dashboard` |
+| ClusterRole | The ClusterRole to assign to the Kubernetes Dashboard. | `read-only` |
+| certDuration | Self-signed certificate duration in hours. | 8760h (365d) |
+| certRenewal | Self-signed certificate renewal in hours | 720h (30d) |
+| enableInsecureLogin | RBAC ensures secure login. | `false` |
+| serviceType | The service type for the dashboard. Supported values are ClusterIP, LoadBalancer, and NodePort. | `ClusterIP` |
+| skipLogin | Enables or disables the skip login option on the dashboard. | `false` |
+| enableInsecureLogin | Enables non-Secure Sockets Layer (SSL) login. Dashboard login is always restricted to HTTP(S) + localhost or HTTPS and external domain. | `false` |
+| ingress:enabled | Ingress configuration to access the ClusterIP, loadBalancer, or nodePort. | `false` |
+
+## Usage
+
+To use the Spectro Kubernetes Dashboard pack, you have to add it to your cluster profile. The Spectro Kubernetes Dashboard pack has default settings for **Access** and **Identity Provider**. These settings automatically install the latest version of the Spectro Proxy pack when you create the cluster profile.
+
+Changing these default settings may require some additional configuration.
+<br /> 
+
+### Access
+
+The default setting is **Private**. If you change the setting to **Public** and your cluster is in a public cloud, there is no additional configuration.  
+
+However, if you change **Access** to **Public** and your cluster is in a private cloud, you have to manually add and configure the Spectro Proxy pack to your cluster profile. For more information, refer to the [Spectro Proxy](/integrations/frp) guide.
+
+<br /> 
+
+### Identity Provider 
+
+The default setting is **Palette**.
+
+<br />
+
+- **Palette**: No configuration is required. This setting makes Palette the IDP, so any user with a Palette account in the tenant and the proper permissions to view and access the project's resources is able to log into the Kubernetes dashboard.
+
+- **Inherit from Organization**: This setting requires you to configure OIDC authentication in Tenant Settings. In Tenant Admin scope, navigate to **Tenant Settings > SSO**, choose **OIDC**, and provide your third-party IDP details. For more information, check out the [SSO Setup](/user-management/saml-sso) guide. 
+
+- **None**: For all cloud services except Azure AKS, this setting requires you to configure OpenID Connect (OIDC) authentication. Azure AKS clusters require authentication through Active Directory (AD).
+
```

Comment:
@MGamya @arvind-sys 
Are descriptions for **Inherit from Organization** and **None** accurate?

Reply:
@ritawatson We don't need to specify the Azure AKS, as we will have different section to list supported clouds to have spectro k8s dashboard integration.

Reply:
@ravi-k8 Removed Azure AKS throughout.